packages/firestore/src/local/shared_client_state.ts

```diff
@@ -102,9 +102,12 @@ export interface SharedClientState {
    */
   addLocalQueryTarget(targetId: TargetId): QueryTargetState;
 
-  /** Removes a Query Target ID for the local Firestore clients. */
+  /** Removes the Query Target ID association from the local client. */
```

Comment:
Can you add "Called by the primary client to ..." ?

Reply:
Done.